pipelines/templates/tasks/upmpackages.yml

```diff
@@ -0,0 +1,18 @@
+# [Template] Builds and publishes UPM .tgz packages
+parameters:
+  version: '$(MRTKVersion)-preview.$(Build.BuildNumber)'
```

Comment:
Do we manually format `Build.BuildNumber` somewhere? Or does ADO give us the default?
This is less a question about packaging and more about how we can update this if we want, or if the default is fine.

Looks like it'll be something like `20200730.1`

Reply:
this is the same mechanism as used by the unitypackage task. it should be passed to us as well.

i _do_ want to make sure we can spin final packages with the official version at some point. i do not believe a simple file rename will be sufficient here.

Reply:
since it's already working now probably don't change it in this iteration, but going forward we shouldn't be putting business logic in the lower level functions like this (i.e. the fact that it's a preview should be somethihng higher up telling it it's a preview, not the code here that's figuring this out).

The fact that it gets interpolated this way is kinda weird - it would be fine, however, to take these in separately and have something (i.e. another field) that is "isPreview' which would trigger this logic.

Reply:
Let me take a look at how to do this better.... I have an idea based on what you said @wiwei

Reply:
separated them :)

a future PR will allow the pipeline to specify official builds (which will not use the build number)